docs/api/browser-window.md

```diff
@@ -222,17 +222,12 @@ It creates a new `BrowserWindow` with native properties as set by the `options`.
       the top left.
     * `hiddenInset` - Results in a hidden title bar with an alternative look
       where the traffic light buttons are slightly more inset from the window edge.
-    * `customButtonsOnHover` Boolean (optional) - Draw custom close,
-      and minimize buttons on macOS frameless windows. These buttons will not display
-      unless hovered over in the top left of the window. These custom buttons prevent
-      issues with mouse events that occur with the standard window toolbar buttons.
-      **Note:** This option is currently experimental.
+    * `customButtonsOnHover` - Results in a hidden title bar and a full size
+      content window, the traffic light buttons will display when being hovered
+      over in the top left of the window.  **Note:** This option is currently
+      experimental.
   * `trafficLightPosition` [Point](structures/point.md) (optional) - Set a
-    custom position for the traffic light buttons. Can only be used with
-    `titleBarStyle` set to `hidden` or `customButtonsOnHover`.
-  * `fullscreenWindowTitle` Boolean (optional) - Shows the title in the
-    title bar in full screen mode on macOS for all `titleBarStyle` options.
-    Default is `false`.
+    custom position for the traffic light buttons in frameless windows.
```

Comment:
This option was used to work around a bug, since the bug is fixed with this PR this option is no longer needed, so removing the option does not break anything.

This is a very special case, not sure how others in @electron/wg-api think. I'm fine marking this as `semver/minor` though.

Reply:
OK that makes sense. My concern was that someone might have this set to `fullscreenWindowTitle: true` and that the app's behavior would change because that setting was no longer present.

If pre-existing code that uses that property either with `true` or `false` just works correctly anyway, and doesn't give any warnings/exceptions about an unrecognized property, then I agree with your call of `semver/patch`.